README.md

```diff
@@ -31,6 +31,7 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
   - [Toggle Button](#toggle-button)
     - [Toggle Button as Checkboxes](#toggle-button-as-checkboxes)
     - [Toggle Button as Radio Button](#toggle-button-as-radio-button)
+    - [Rounded Buttons](#rounded-buttons)
```

Comment:
Please indent this line correctly, as Rounded Buttons is not a subsection of Toggle Button